packages/core/src/Constants/Hints.ts

```diff
@@ -0,0 +1,48 @@
+import { type SignPage } from "./Titles";
+
+type HintData = {
+  header?: string;
+  description?: string;
+};
+const basicOperationHeader = undefined;
+const basicOperationDescription = undefined;
+
+const finalizationConsequences =
+  "Finalized funds are returned to your balance, allowing you to spend them or stake them again to earn rewards. Delegation rewards for these funds stop once they are finalized.";
+const unstakingConsequences = `The unstaking process takes 4 cycles (~10 days) to unlock your staked funds, making them finalizable on request. You will receive delegation rewards for finalizable funds. ${finalizationConsequences}`;
```

Comment:
```suggestion
const unstakingConsequences = `Unstaked tez immediately stop accruing staking rewards, but sill give rights to delegation rewards. Unstaked tez remain locked for up to 4 cycles (~11 days), after which they can be finalized on request. ${finalizationConsequences}`;
```
If we wanted to be less contrieved : an unstake request on cycle n because finalizable at the beginning of cycle n+4. This means we need to wait at least 3 full cycles, and up to 4.

See the diagram here

https://docs.tezos.com/using/staking#staking-periods-and-delays